test/paper-input-error.html

```diff
@@ -31,7 +31,7 @@
 
   <test-fixture id="auto-validate-numbers">
     <template>
-      <paper-input-container auto-validate>
+      <paper-input-container auto-validate attr-for-value="bind-value">
```

Comment:
Yeah, it's on purpose. The textarea is using the `input` event to update the value. It looks like `bind-value` didn't fire for some reason but have to debug in the other repository.